### PR TITLE
samples: bluetooth: nrf_dm: Add Zero-distance calibration note

### PR DESCRIPTION
**Pull request by maje-emb**
Adds zero distance calibration information to sample documentation.

### CODE REVIEW

File: samples/bluetooth/nrf_dm/README.rst [hunk 1]
```diff
@@ -78,6 +78,16 @@ The addresses of these devices are stored in a list.
 When a device does not respond within a certain period of time, it is removed from the list.
 Also the last measurement results with the peers are stored in this list.
 
+Zero-distance calibration
+*************************
+
+The measured values of distance measurement include an off-set.
```
Comment (eriksandgren) [inline samples/bluetooth/nrf_dm/README.rst:84]: I think it is usually written "offset" without the hyphen

File: samples/bluetooth/nrf_dm/README.rst [hunk 2]
```diff
@@ -78,6 +78,16 @@ The addresses of these devices are stored in a list.
...
+The off-set value may be due to the design of the radio circuit, the antenna used or the designed PCB Layout.
+Zero-distance value should be handled outside the library and sample.
```
Comment (eriksandgren) [inline samples/bluetooth/nrf_dm/README.rst:86]:
```suggestion
Zero-distance offset should be compensated for by the user of the library and sample.
```

File: samples/bluetooth/nrf_dm/README.rst [hunk 3]
```diff
@@ -78,6 +78,16 @@ The addresses of these devices are stored in a list.
...
+You should manually perform the calibration.
+To determine the off-set value, place the devices at reference distance and collect a batch of measurement data.
```
Comment (eriksandgren) [inline samples/bluetooth/nrf_dm/README.rst:88]: Should we mention the suggested distance here, I think Carsten mentioned 60 cm?

File: samples/bluetooth/nrf_dm/README.rst [hunk 4]
```diff
@@ -78,6 +78,16 @@ The addresses of these devices are stored in a list.
 When a device does not respond within a certain period of time, it is removed from the list.
 Also the last measurement results with the peers are stored in this list.
 
+Zero-distance calibration
+*************************
+
+The measured values of distance measurement include an off-set.
+The off-set value may be due to the design of the radio circuit, the antenna used or the designed PCB Layout.
+Zero-distance value should be handled outside the library and sample.
+You should manually perform the calibration.
```
Comment (eriksandgren) [inline samples/bluetooth/nrf_dm/README.rst:87]:
```suggestion
Users of the library need to manually perform the calibration.
```

**Review by eriksandgren — COMMENTED** [review]
Just some suggestions

File: samples/bluetooth/nrf_dm/README.rst (hunk 1 above)
Comment (maje-emb) [reply samples/bluetooth/nrf_dm/README.rst:84]: Fixed.

File: samples/bluetooth/nrf_dm/README.rst (hunk 3 above)
Comment (maje-emb) [reply samples/bluetooth/nrf_dm/README.rst]: Good point. Added a note about the suggested distance.

File: samples/bluetooth/nrf_dm/README.rst [hunk 5]
```diff
@@ -78,6 +78,16 @@ The addresses of these devices are stored in a list.
 When a device does not respond within a certain period of time, it is removed from the list.
 Also the last measurement results with the peers are stored in this list.
 
+Zero-distance calibration
+*************************
+
+The measured values of distance measurement include an offset.
+The offset value may be due to the design of the radio circuit, the antenna used or the designed PCB Layout.
```
Comment (peknis) [inline samples/bluetooth/nrf_dm/README.rst:85]:
```suggestion
The offset value may depend on the design of the radio circuit, the antenna used, or the PCB layout.
```

File: samples/bluetooth/nrf_dm/README.rst [hunk 6]
```diff
@@ -78,6 +78,16 @@ The addresses of these devices are stored in a list.
...
+Zero-distance offset should be compensated for by the user of the library and sample.
```
Comment (peknis) [inline samples/bluetooth/nrf_dm/README.rst:86]:
```suggestion
When using a library and a sample, you need to compensate the zero-distance offset and perform the calibration manually.
```

File: samples/bluetooth/nrf_dm/README.rst [hunk 7]
```diff
@@ -78,6 +78,16 @@ The addresses of these devices are stored in a list.
...
+Users of the library need to manually perform the calibration.
```
Comment (peknis) [inline samples/bluetooth/nrf_dm/README.rst:87]:
```suggestion
```
Okay to combine this with what was said below, as I'm suggesting? Does it change the meaning?

File: samples/bluetooth/nrf_dm/README.rst [hunk 8]
```diff
@@ -78,6 +78,16 @@ The addresses of these devices are stored in a list.
...
+To determine the offset value, place the devices at reference distance, the suggested distance is 60cm and collect a batch of measurement data.
```
Comment (peknis) [inline samples/bluetooth/nrf_dm/README.rst:88]:
```suggestion
To determine the offset value, place the devices at a reference distance of 60 cm from each other and collect a batch of measurement data.
```

**Review by peknis — CHANGES_REQUESTED** [review]

File: samples/bluetooth/nrf_dm/README.rst (hunk 7 above)
Comment (maje-emb) [reply samples/bluetooth/nrf_dm/README.rst]: I think this sentence is not needed.

File: samples/bluetooth/nrf_dm/README.rst [hunk 9]
```diff
@@ -78,6 +78,15 @@ The addresses of these devices are stored in a list.
...
 Also the last measurement results with the peers are stored in this list.
 
+Zero-distance calibration
+*************************
+
+The measured values of distance measurement include an offset.
+The offset value may depend on the design of the radio circuit, the antenna used, or the PCB layout.
+When using a library and a sample, you need to compensate the zero-distance offset and perform the calibration manually.
+To determine the offset value, place the devices at a reference distance of 60 cm from each other and collect a batch of measurement data.
+The difference between the averaged measurement value and the actual distance is the offset.
```
Comment (wulffern) [inline samples/bluetooth/nrf_dm/README.rst:88]: I would say "The difference between the averaged measurement value and the actual distance can be used as an estimate of the offset."

Comment (maje-emb) [reply samples/bluetooth/nrf_dm/README.rst]: ACK

File: samples/bluetooth/nrf_dm/README.rst [hunk 10]
```diff
@@ -78,6 +78,15 @@ The addresses of these devices are stored in a list.
...
 Also the last measurement results with the peers are stored in this list.
 
+Zero-distance calibration
+*************************
+
+The measured values of distance measurement include an offset.
+The offset value may depend on the design of the radio circuit, the antenna used, or the PCB layout.
+When using a library and a sample, you need to compensate the zero-distance offset and perform the calibration manually.
+To determine the offset value, place the devices at a reference distance of 60 cm from each other and collect a batch of measurement data.
+The difference between the averaged measurement value and the actual distance can be used as an estimate of the offset.
```
Comment (peknis) [inline samples/bluetooth/nrf_dm/README.rst:88]:
```suggestion
The difference between the average measurement value and the actual distance can be used as an estimate of the offset.
```

**Review by peknis — APPROVED** [review]
Approved with a minor nit.

File: samples/bluetooth/nrf_dm/README.rst (hunk 10 above)
Comment (maje-emb) [reply samples/bluetooth/nrf_dm/README.rst]: Fixed.

**Review by KAGA164 — APPROVED** [review]

**Review by eriksandgren — APPROVED** [review]

**Pull request merged**